packages/react-dom/package.json

```diff
@@ -17,7 +17,7 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@esm-bundle/react-dom": "17.0.1"
+    "@esm-bundle/react-dom": "16.14.0"
```

Comment:
This is the final v16 release. We can get this published and then renovate should update us back to v17 automatically